ui/src/logs/actions/index.ts

```diff
@@ -489,8 +522,8 @@ export const executeHistogramQueryAsync = () => async (
 
 export const executeTableQueryAsync = () => async (dispatch): Promise<void> => {
   await Promise.all([
-    dispatch(executeTableForwardQueryAsync()),
-    dispatch(executeTableBackwardQueryAsync()),
+    dispatch(executeTableNewerQueryAsync()),
```

Comment:
would Results be appropriate within this func name for consistency with `executeTableOlderResultsQueryAsync`?